lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -38,15 +38,30 @@
 
                 <p class="accomplishment-statement-detail copy copy-lead">${accomplishment_copy_course_description}</p>
             </div>
+
+            % if certificate_data and certificate_data.get('org_logo_path', ''):
```

Comment:
NOTE: for print layout purposes (where orgs are side by side with the accomplishment statement), I've moved this up in the DOM before signatories.